src/pages/Cart/Count/Count.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import './Count.scss'
+
+const Count = ({ id, quantity, onChange }) => {
+  const handleCount = value => {
+    if (quantity + value < 1) return
+    onChange(quantity + value)
+  }
+
+  return (
+    <div className="count" id={id}>
```

Comment:
```suggestion
    <div className="count">
```
id 속성은 쓰이지 않을 것 같아서 제거하셔도 좋습니다!

Reply:
여기 아직 반영이 안됐네요!